ambry-api/src/main/java/com/github/ambry/clustermap/ClusterParticipant.java

```diff
@@ -14,6 +14,7 @@
 
 package com.github.ambry.clustermap;
 
+import com.github.ambry.router.Callback;
```

Comment:
Looks like you imported a different type of Callback (should be account service callback not router callback).

Reply:
the router callback is the interface, and I implement with account service callback. Let me know if I misunderstanding something.

Reply:
After discussion, I moved the callback interface to commons dir which is more reasonable.